app/models/gravatar.rb

```diff
@@ -0,0 +1,36 @@
+class Gravatar
```

Comment:
Shall we move this to `lib/`?  There are of course different schools about where to put what. Personally, I like `lib` to be a place that has no ties to the domain of the app and could be used somewhere else without changes. A gravatar generator qualifies for this.

Reply:
Will move it to `lib/`